lib/pf/Switch/Ruckus.pm

```diff
@@ -113,12 +115,18 @@ sub parseTrap {
     my $trapHashRef;
     my $logger = $self->logger;
 
-    $logger->debug("trap currently not handled.  TrapString was: $trapString");
-    $trapHashRef->{'trapType'} = 'unknown';
-
+    # Handle WIPS Trap
+    if ( $trapString =~ /\.1\.3\.6\.1\.4\.1\.25053\.2\.2\.2\.20 = STRING: \"([a-f0-9]{2}:[a-f0-9]{2}:[a-f0-9]{2}:[a-f0-9]{2}:[a-f0-9]{2}:[a-f0-9]{2})/ ) {
```

Comment:
Fine to delete. I got it form an older bit of code for ruckus integration. Can be deleted. I can amend the commit if needed.